### PR TITLE
RV modification functions

### PR DESCRIPTION
**Pull request by lrennels**
#795

### CODE REVIEW

**Comment by lrennels** [comment]
@grantmcdermott this should help with some of the issues you ran into here: https://github.com/anthofflab/MimiPAGE2009.jl/issues/222

**Comment by grantmcdermott** [comment]
Super, thanks for the head up @lrennels!

Two minor aside, which I hope you don't mind me mentioning here although they're only tangentially related to this PR.
- Assuming this is still the case, could you mention somewhere in the documentation that `EmpiricalDistribution` isn't export by default when loading Mimi?
- Talking documentation, is there a reason why "Documentation" isn't a main sub-menu link on the left-hand side of the Mimi [website](https://www.mimiframework.org/)?

**Comment by lrennels** [comment]
@grantmcdermott I don't mind at all, feedback is really helpful. Per your first comment yes that's a good idea I'll add some documentation on the status of `EmpiricalDistribution`, need to go look at if there's a reason we haven't documented it (i.e. it's in Beta format and we don't want to throw users in the deep end unless they're comfortable with it), or we just hadn't done it yet.  I think I didn't put it in the sub-menu link since those are mostly links to pages within the forum as opposed to external links, but if you think it would be helpful I could put it there!  I'm not sure the left hand side label can be a link, but it could link to a page that provides the link.

**Comment by lrennels** [comment]
@grantmcdermott just added a documentation link to the left-side bar, good idea!

File: wip/temp.jl [hunk 1]
```diff
@@ -0,0 +1,34 @@
+using Mimi
```
Comment (arnavgautam) [inline wip/temp.jl:1]: Was this file intended to be checked in? I can't see how it connects to the rest of the files in this PR

**Review by arnavgautam — DISMISSED** [review]
Always exciting to see us surfacing more details on errors

File: wip/temp.jl (hunk 1 above)
Comment (lrennels) [reply wip/temp.jl]: Ah good point thank you, deleted it!

**Review by arnavgautam — APPROVED** [review]

**Pull request merged**